.gitignore

```diff
@@ -69,3 +69,6 @@ test/bpf/_results
 
 # generated from make targets
 *.ok
+
+# Documentation
+Documentation/.config/
```

Comment:
Why is this needed? What writes this directory?

Reply:
It's unrelated to the PR itself, but I noticed that this dir gets created after `make -C Documentation`.

Reply:
I don't get that directory after running that command. Maybe something related to your environment? Could that be related to podman?

Reply:
cc @errordeveloper

Reply:
After rebasing with master that file is gone for me, both for docker and podman... weird.